addon/serializers/github-release.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import GithubSerializer from './github';
+
+export default GithubSerializer.extend({
+  keyForAttribute(attr) {
+    return Ember.String.decamelize(attr);
```

Comment:
I would prefer importing the shim rather than all of Ember

https://github.com/ember-cli/ember-cli-shims/blob/master/app-shims.js#L185